objcts.js

```diff
@@ -0,0 +1,9 @@
+for(i=1; i<=15;i++){
+let k = i;
+let j = k*k;
+let numbers ={
+[k] : [j],
```

Comment:
This Isn't what we want. Computed properties should only be used for object keys i.e the correct version of this line would be
```js
let numbers = {
// use [] in the key because we want to use the value of the variable number as the key
[k]: j
}
```
> But we don't need it since we should create the object once outside the loop